padl/transforms.py

```diff
@@ -1798,3 +1851,22 @@ def group(transform: Union[Rollout, Parallel]):
     2-tuple whose first item will be passed to `a` and whose second item will be passed to `b + c`.
     """
     return transform.grouped()
+
+
+class _ItemGetter:
```

Comment:
Add back the docs from the previous method
```python
    """A simple dataset.
    Takes *samples* and applies *transform* to it.
    :param samples: An object implementing __getitem__ and __len__.
    :param transform: Preprocessing transform.
    :param exception: Exception to catch for (fall back to *default*).
    :param default: The default value to fall back to in case of exception.
    """
```